shared/actions/kbfs/platform.desktop.js

```diff
@@ -32,30 +32,33 @@ export function platformOpenInKBFS (path: string = Constants.defaultKBFSPath): A
 
     // On windows the path isn't /keybase
     // We can figure it out by looking at the extendedConfig though
-    if (process.platform === 'win32' && kbfsPath === Constants.defaultKBFSPath) {
-      const extendedConfig = Promise.resolve(state.config.extendedConfig)
+    if (process.platform === 'win32') {
+      if (kbfsPath === Constants.defaultKBFSPath) {
+        const extendedConfig = Promise.resolve(state.config.extendedConfig)
 
-      extendedConfig.then(extendedConfig => {
-        const kbfsClients = extendedConfig.Clients.filter(c => c.clientType === Common.ClientType.kbfs)
-        if (kbfsClients.length !== 1) {
-          return Promise.reject("There isn't exactly one kbfs client")
-        }
+        extendedConfig.then(extendedConfig => {
+          const kbfsClients = extendedConfig.Clients.filter(c => c.clientType === Common.ClientType.kbfs)
+          if (kbfsClients.length !== 1) {
+            return Promise.reject("There isn't exactly one kbfs client")
+          }
 
-        // Hacky Regex to find a mount point on windows matches anything like foobar:\ or K:\
-        const kbfsPath = kbfsClients[0].argv.filter(arg => arg.search(/.*:\\?$/) === 0)[0]
+          // Hacky Regex to find a mount point on windows matches anything like foobar:\ or K:\
+          const kbfsPath = kbfsClients[0].argv.filter(arg => arg.search(/.*:\\?$/) === 0)[0]
 
-        if (!kbfsPath) {
-          return Promise.reject("Couldn't figure out kbfs path from argv")
-        }
+          if (!kbfsPath) {
+            return Promise.reject("Couldn't figure out kbfs path from argv")
+          }
 
-        return Promise.resolve(kbfsPath + '\\')
-      }).then(kbfsPath => {
-        dispatch({type: Constants.changeKBFSPath, payload: {path: kbfsPath}})
+          return Promise.resolve(kbfsPath + '\\')
+        }).then(kbfsPath => {
+          dispatch({type: Constants.changeKBFSPath, payload: {path: kbfsPath}})
+          open(kbfsPath, path)
+        }).catch(e => {
+          console.warn('Error in parsing kbfsPath: ', e)
+        })
+      } else {
```

Comment:
i don't think its worked for awhile reliably. you can get this to work once (and it'll try and open it twice)